### PR TITLE
Whois integration connectivity issue

### PR DESCRIPTION
**Pull request by iyeshaya**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/40401

## Description
Fixed a bug where function gets the wrong params
## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/Whois/Integrations/Whois/Whois_test.py [hunk 1]
```diff
@@ -201,3 +201,20 @@ def test_ip_command(mocker):
...
+        - proxy address
+
+    When:
+        - running the get_whois_ip function
+
+    Then:
+        - Verify the function doesn't fail due to type errors
+    """
+    from Whois import get_whois_ip
+    mocker.patch.object(demisto, 'params', return_value={"proxy": "https://test.com"})
```
Comment (Itay4) [inline Packs/Whois/Integrations/Whois/Whois_test.py:218]: shouldnt you mock `proxy_url` here?

Comment (iyeshaya) [reply Packs/Whois/Integrations/Whois/Whois_test.py:218]: Not really, 
we test for type mismatch when trying to connect to the proxy server, it doesn't matter if the proxy server is valid.

Comment (Itay4) [reply Packs/Whois/Integrations/Whois/Whois_test.py:218]: didnt mean the url itself, i mean the param you're currently mocking is the boolean one and not the string one which is used for the `ip` command

File: Packs/Whois/Integrations/Whois/Whois.yml [hunk 2]
```diff
@@ -8,12 +8,13 @@ configuration:
   required: false
   type: 8
 - display: Proxy URL
-  additionalinfo: "Domain and Whois commands: supports socks4/socks5/http connect\
-    \ proxies (e.g. socks5h://host:1080)."
+  additionalinfo: "Domain and Whois commands: Supports socks4/socks5/http connect proxies (e.g. socks5h://host:1080).
```
Comment (Itay4) [inline Packs/Whois/Integrations/Whois/Whois.yml:11]:
```suggestion
  additionalinfo: "Supports socks4/socks5/http connect proxies (e.g. socks5h://host:1080).
```

**Review by Itay4 — APPROVED** [review]

**Pull request merged**